components/odh-notebook-controller/controllers/notebook_oauth.go

```diff
@@ -42,9 +42,9 @@ const (
 	OAuthServicePort     = 443
 	OAuthServicePortName = "oauth-proxy"
 	// OAuthProxyImage uses sha256 manifest list digest value of v4.14 image for AMD64 as default to be compatible with imagePullPolicy: IfNotPresent, overridable
```

Comment:
@coderabbitai please create github issue for this in the current github repository

45-47: Nit: Clean up adjacent comment to avoid stale/ambiguous wording.

The preceding Line 44 still mentions “v4.14 image for AMD64,” which is misleading now that we use the RHEL9 image (likely a multi-arch manifest list). Consider updating Line 44 to be version/arch-agnostic.

Suggested replacement for Line 44:

// OAuthProxyImage uses a sha256 manifest-list digest as the default to be compatible with imagePullPolicy: IfNotPresent; overridable

Reply:
@coderabbitai you created the issue in the upstream kubeflow/kubeflow repository again. This is wrong and don't do it ever again! We want any issues created for this repo to be tracked in this repository, which means opendatahub-io/kubeflow. Please, create the issue again but this time in this repository!